scripts/createComponent.js

```diff
@@ -159,4 +183,8 @@ inquirer
     writeFile(componentOut);
     writeFile(componentTest);
     writeFile(componentIndex);
+
+    if (collection !== 'utils') {
```

Comment:
Do we need `utils` option? I don't think we ever used that.

Reply:
Yeah, now it is broken.. lets remove it from options

Reply:
It's broken?

Reply:
Well, I tried, it failed, so... Unless I missed something.

Reply:
utils is gone 💥🤯

Reply:
Sad doesnt work with subDirs, but.. it is ok for now, I guess
![image](https://user-images.githubusercontent.com/81580355/116883909-cb0c8300-ac2e-11eb-94ea-a55c5522439b.png)

Reply:
😄  Yeah we removed that in https://github.com/iTwin/iTwinUI-react/commit/2975e1a480afa07f141d0868dfcf691952a75578